extension/chrome/elements/composer/composer-errs.ts

```diff
@@ -25,7 +25,7 @@ export class ComposerErrs extends ComposerComponent {
 
   private debugId = Str.sloppyRandom();
 
-  initActions = () => {
+  public initActions = () => {
```

Comment:
@tomholub, maybe, it would be good to make `initActions` optional method?
What do you think?

Reply:
I'd keep it required, there should be relatively few that don't actually have it declared, and I think it's good to have a place where the handlers would go if needed.

Reply:
But I'll probably eventually refactor it like view modules one day

Reply:
Got it, thanks!